tests/acceptance/test_sro_measures_dsl.py

```diff
@@ -0,0 +1,157 @@
+from databuilder import Measure
+from databuilder.tables import clinical_events, practice_registrations
+
+from .codelists import (
+    alt_codelist,
+    asthma_codelist,
+    cholesterol_codelist,
+    copd_codelist,
+    hba1c_codelist,
+    medication_review_codelist,
+    qrisk_codelist,
+    rbc_codelist,
+    sodium_codelist,
+    systolic_bp_codelist,
+    tsh_codelist,
+)
+
+
+def test_sro_measures_study(cohort, subtest):
+    index_date = "2020-11-01"
+    end_of_index_month = "2020-11-30"
```

Comment:
Can we do this with date arithmetic?

Reply:
Would you do this using Data Builder or standard Python? The non-DSL test uses standard Python:

https://github.com/opensafely-core/databuilder/blob/884e7d38ec02e58da9c709b09aeea486b495f607/tests/acceptance/test_sro_measures_study.py#L45-L47

Do we have functionality in Data Builder for this?

Reply:
I suppose it would be Python, so it doesn't really gain us anything. As you were.

Reply:
Aye aye, 🙂